app/models/contact.rb

```diff
@@ -0,0 +1,9 @@
+class Contact < ApplicationRecord
+  belongs_to :user
+  has_many :interactions, dependent: :destroy
+
+  validates :name, :user_id, :phone_number, :relationship, :address, presence: true
+  validates :email, uniqueness: true
+  validates :email, format: { with: URI::MailTo::EMAIL_REGEXP, message: "Must be a valid email address 📪" }
+  validates :relationship, inclusion: { in: %w(friend family colleague), message: "%{value.capitalize} is not a valid relationship 🕺" }
```

Comment:
Should we have this as a constant at the top of the class? `%w(friend family colleague)`

Reply:
Implemented 🌈